### PR TITLE
chore: upgrade to Rust 1.74.0

### PR DESCRIPTION
**Pull request by iuioiua**

### CODE REVIEW

File: lib/wasmbuild.generated.js [hunk 1]
```diff
@@ -222,8 +222,8 @@ const imports = {
   },
 };
 
-import { Loader } from "../loader.ts";
-import { cacheToLocalDir } from "../cache.ts";
+import { Loader } from "file:///Users/asher/GitHub/wasmbuild/loader.ts";
```
Comment (sigmaSd) [inline lib/wasmbuild.generated.js:225]: this needs to be reverted

Comment (iuioiua) [reply lib/wasmbuild.generated.js]: Whoops. Fixed. Thanks!

**Comment by iuioiua** [comment]
PTAL, @dsherret

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks!

**Pull request merged**